gef.py

```diff
@@ -10647,6 +10686,26 @@ def invoke(self, args, from_tty):
         gdb.execute("gef help")
         return
 
+    def add_context_pane(self, pane_name, pane_function):
+        """Add a new context pane to ContextCommand"""
+        context_obj = None  # type: ContextCommand
+        for cmd, class_name, class_obj in self.loaded_commands:
+            if isinstance(class_obj, ContextCommand):
+                context_obj = class_obj
+                break
+
+        if not context_obj:
```

Comment:
I tried really hard but couldn't find a case where context object cannot be found with having gef collapsing entirely. This check is not necessary.